mautic/mautic
# Update ddev configuration

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

DDEV has a [builtin feature](https://twitter.com/randyfay/status/1713897478364279272?t=AdJfKrJViDjPNYH6q1u2vw&s=09) which allows you to update the configuration file automatically to take into account changes that have come into the code recently.

I ran this on our code base we haven't really updated our configuration file in quite some time, and this was the result.  I have left in the comments in the file because this may be helpful for people when they are using DDEV locally and need to make some small tweaks or configure specific settings.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make sure that DDEV is working correctly
